src/test/deploy/functions/runtimes/discovery/v1alpha1.spec.ts

```diff
@@ -307,6 +307,63 @@ describe("buildFromV1Alpha", () => {
       await expect(resolveBackend(parsed)).to.eventually.deep.equal(expectedBackend);
     });
 
+    it("copies v1 schedules", async () => {
```

Comment:
Can we add 2 test cases? One that uses *Duration another that uses *Seconds?

Reply:
Let's do this in a follow up PR.

Reply:
There's already a test that uses *Seconds on L367